Gems/ROS2/Code/Source/Communication/TopicConfiguration.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include "ROS2/Communication/TopicConfiguration.h"
+#include "ROS2/Utilities/ROS2Names.h"
```

Comment:
For these includes angle brackets should be used: https://github.com/o3de/sig-core/blob/main/governance/Coding-Standards-and-Style-Guide.md#including-headers